docs/proposals/policy-engine.md

```diff
@@ -0,0 +1,49 @@
+# Policy Engine - Proposal
+
+## Goals
+
+The management of the evolution and maintainability of semantic conventions and
+application telemetry schemas relies on rules or policies that must be verified
+before publishing a new version. The goal of these policies is to ensure the
+coherence and quality of the conventions and schemas over the long term. Below
+are some examples of policies which are already verified or that could be
+verified:
```

Comment:
I agree it would be great to have this automated and described independently of tooling (weaver or build-tools).

We already have a quite exhaustive textual description of policies [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#semantic-conventions-stability).

Reply:
I will add an example of rego rules to represent some of these policies.